ui/app/components/secret-engine/configure-aws.hbs

```diff
@@ -56,7 +56,7 @@
     {{/if}}
     {{#if (eq this.accessType "wif")}}
       {{! WIF Fields }}
-      {{#each @issuerConfig.attrs as |attr|}}
+      {{#each @issuerConfig.displayAttrs as |attr|}}
```

Comment:
Nice! I think this reads nicely and makes figuring out what these attrs in the model are much easier! 🤩